src/sardana/util/graphics.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+##############################################################################
+##
+# This file is part of Sardana
+##
+# http://www.sardana-controls.org/
+##
+# Copyright 2011 CELLS / ALBA Synchrotron, Bellaterra, Spain
+##
+# Sardana is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+##
+# Sardana is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+##
+# You should have received a copy of the GNU Lesser General Public License
+# along with Sardana.  If not, see <http://www.gnu.org/licenses/>.
+##
+##############################################################################
+
+"""This package provides graphics related helper functions"""
+
+import os
+import sys
+
+
+def xsession_available():
+    """
+    Checks if an X-session is available.
```

Comment:
This should say that it assumes that display is always available if the system isn't linux.

Reply:
I have no idea how this works on windows so there I need someone to tell me how this is done on windows.
I made the assumption that on windows you can not do any remote starting of sardana as it would be running locally. I assume you will always have an RDP or a VNC session, thus always having access to a window manager. Or are you running some headless windows machines that run sardana?

I excluded OSX since that is not in the scope of the project.

Reply:
I don't think windows without a graphical display is a thing, so there is no need for any check. Just the docstring should mention that so nobody gets confused.